src/project_file_walker.rs

```diff
@@ -95,10 +94,10 @@ pub(crate) fn get_project_file_walker_types(language: Option<SupportedLanguage>)
     let mut types_builder = TypesBuilder::new();
     types_builder.add_defaults();
     if let Some(language) = language {
-        types_builder.select(get_supported_language_name_for_ignore_select(language));
+        types_builder.select(language.name_for_ignore_select());
     } else {
-        for name_for_ignore_select in SUPPORTED_LANGUAGE_NAMES_FOR_IGNORE_SELECT.iter() {
```

Comment:
The top-level `SUPPORTED_LANGUAGE_LANGUAGES`/`SUPPORTED_LANGUAGE_NAMES_FOR_IGNORE_SELECT` can really be considered "private"/"implementation details" (and I did in fact make them not `pub`) since you can achieve eg here iterating over them by iterating over the main `ALL_SUPPORTED_LANGUAGES` and then accessing the corresponding relevant "field" via the helper methods (eg here `.name_for_ignore_select()`)